### ISSUE TITLE
Load Game Bug

### ISSUE THREAD

**Issue by Automa-C3PO** [JACoders/OpenJK#1087]
Another issue with SP OpenJK. This time I restarted all the variables, but the bug persists. In many missions, especially after _tier2_, every time I load a savegame or quicksave, the game scripts like freeze and I can't go on...during this freeze, most of coded features such as levers and buttons, elevators, big tanked charges (where appliable), key deaths, checkpoints, and scripted objects, just won't work. It's like everything is stuck on a code level, preventing the mission from ending or unlocking other stages...the only way to unblock it is to start over again. Would you know what causes this?
Looks similar to a bug someone else had years ago:
https://github.com/JACoders/OpenJK/issues/990#issue-372294941

**Comment by Automa-C3PO** [comment]
I made tests with latest releases, it seems that the post-load bug was caused by scrambled files, I restored the main _jagamex86.dll_ from Dusty's patch with the original in the build, now it loads again with all scripts and triggers active, and working checkpoints. So everyone who had this should not accidentally merge libraries from other mods inside OpenJK folder.
